domains/game/Game.jsx

```diff
@@ -201,6 +225,44 @@ const Game = ({ playerName, game }) => {
                     <Text>Replace pucks</Text>
                 </TouchableOpacity>
             ) : null}
+
+            <Modal
+                animationType="slide"
+                visible={modalVisible}
+                transparent={true}
+                onRequestClose={() => {
+                    setModalVisible(!modalVisible);
+                }}
+            >
+                <View style={styles.modalView}>
+                    <Pressable
+                        style={[styles.button, styles.buttonClose]}
+                        onPress={() => closeAdvice()}
```

Comment:
you can pass closeAdvice directly, now need to create a function that does nothing more